### ISSUE TITLE
Private Azure bucket as backend

### ISSUE THREAD

**Issue by abhishekvasu94** [NVIDIA/aistore#81]
Hi,

I have my datasets stored in a private Azure blob storage, and I was wondering if I could use that as my backend bucket for the AIS bucket?

Thanks

**Comment by abhishekvasu94** [comment]
Thanks for the quick reply. Unfortunately, the problem is not yet resolved. I have set environment variables for `AZURE_STORAGE_ACCOUNT `, `AZURE_STORAGE_KEY `, `AZURE_STORAGE_CONNECTION_STRING`, and `AZURE_STORAGE_SAS_TOKEN`. 

I used the command `ais bucket props set ais://abc backend_bck.name=az://test-bucket` to set the backend bucket, and the request times out - `Head "http://127.0.0.1:10084/v1/buckets/test-bucket?provider=azure": context deadline exceeded (Client.Timeout exceeded while awaiting headers). To show bucket properties, run "ais show bucket BUCKET -v"`. This also happens when I try to list the contents of the remote bucket with ais - `ais bucket ls az://test-bucket`. 

The only way I have been successful in accessing the contents of my Azure blob storage through `ais` is by downloading files to the AIStore bucket. I use the command `ais job start download "https://dtest.blob.core.windows.net/test-bucket/aistore_test_imgs/extended-crowdsourced-images-2.tar.gz$AZURE_STORAGE_SAS_TOKEN" ais://abc`. Most notable in this command is that I had to pass the SAS token with the rest of the url, despite all the authentication info being stored as environment variables. 

Thank you once again for taking a look at this problem :)
